some tests are failing due to the new 'default' argument in `get_prop`

Since fixing #16 4 tests are failing due to `an unexpected keyword argument 'default'`.
 
```
===================================================================================================== FAILURES =====================================================================================================
_______________________________________________________________________________________________ test_parse_exception _______________________________________________________________________________________________
tests/test_argparser.py:13: in test_parse_exception
    argparser = ArgParser(["test", "abc", "123"])
source/dapsenv/argparser.py:40: in __init__
    self.addDaemonCommand()
source/dapsenv/argparser.py:111: in addDaemonCommand
    logmanager_ip = configmanager.get_prop("logmanager_ip", default='127.0.0.1')
E   TypeError: <lambda>() got an unexpected keyword argument 'default'
________________________________________________________________________________________________ test_parse_success ________________________________________________________________________________________________
tests/test_argparser.py:22: in test_parse_success
    argparser = ArgParser(["config", "--global", "--property", "test"])
source/dapsenv/argparser.py:40: in __init__
    self.addDaemonCommand()
source/dapsenv/argparser.py:111: in addDaemonCommand
    logmanager_ip = configmanager.get_prop("logmanager_ip", default='127.0.0.1')
E   TypeError: <lambda>() got an unexpected keyword argument 'default'
_______________________________________________________________________________________________ test_return_version ________________________________________________________________________________________________
tests/test_argparser.py:33: in test_return_version
    argparser = ArgParser(["--version"])
source/dapsenv/argparser.py:40: in __init__
    self.addDaemonCommand()
source/dapsenv/argparser.py:111: in addDaemonCommand
    logmanager_ip = configmanager.get_prop("logmanager_ip", default='127.0.0.1')
E   TypeError: <lambda>() got an unexpected keyword argument 'default'
____________________________________________________________________________________________________ test_main _____________________________________________________________________________________________________
tests/test_init.py:11: in test_main
    main([""])
source/dapsenv/__init__.py:49: in main
    argparser = ArgParser(args)
source/dapsenv/argparser.py:40: in __init__
    self.addDaemonCommand()
source/dapsenv/argparser.py:111: in addDaemonCommand
    logmanager_ip = configmanager.get_prop("logmanager_ip", default='127.0.0.1')
E   TypeError: <lambda>() got an unexpected keyword argument 'default'

```

--- a/tests/test_argparser.py
+++ b/tests/test_argparser.py
@@ -5,10 +5,12 @@ from dapsenv.argparser import ArgParser
 from dapsenv.exitcodes import E_INVALID_CLI
 from dapsenv.general import __version__
 
+def getprop(prop, config_type="", config_path="", default=None):
+    return "0"
 
 # it exits with exit code 2 if an invalid action/sub-command was specified
 def test_parse_exception(monkeypatch):
-    monkeypatch.setattr(configmanager, 'get_prop', lambda _: "fake")
+    monkeypatch.setattr(configmanager, 'get_prop', getprop)
     with pytest.raises(SystemExit) as error:
         argparser = ArgParser(["test", "abc", "123"])
         parsed_args = argparser.parse()
@@ -18,7 +20,7 @@ def test_parse_exception(monkeypatch):
 
 # it succeeds if a valid command was parsed to the command line
 def test_parse_success(monkeypatch):
-    monkeypatch.setattr(configmanager, 'get_prop', lambda _: "fake")
+    monkeypatch.setattr(configmanager, 'get_prop', getprop)
     argparser = ArgParser(["config", "--global", "--property", "test"])
     args = argparser.parse()
     assert args['action'] == 'config'
@@ -28,7 +30,7 @@ def test_parse_success(monkeypatch):
 
 # it returns the correct version number
 def test_return_version(monkeypatch, capsys):
-    monkeypatch.setattr(configmanager, 'get_prop', lambda _: "fake")
+    monkeypatch.setattr(configmanager, 'get_prop', getprop)
     with pytest.raises(SystemExit) as error:
         argparser = ArgParser(["--version"])
         parsed_args = argparser.parse()

--- a/tests/test_init.py
+++ b/tests/test_init.py
@@ -3,10 +3,12 @@ from dapsenv import main, execute, configmanager
 from dapsenv.exceptions import InvalidActionException
 from dapsenv.exitcodes import E_INVALID_CLI, E_NO_IMPLEMENTATION_FOUND
 
+def getprop(prop, config_type="", config_path="", default=None):
+    return "0"
 
 # it fails when no sub-command/action was specified
 def test_main(monkeypatch):
-    monkeypatch.setattr(configmanager, 'get_prop', lambda _: "fake")
+    monkeypatch.setattr(configmanager, 'get_prop', getprop)
     with pytest.raises(SystemExit) as error:
         main([""])
     assert error.value.code == E_INVALID_CLI
@@ -14,7 +16,7 @@ def test_main(monkeypatch):
 
 # it fails if an action is not implemented
 def test_action_not_implemented(monkeypatch):
-    monkeypatch.setattr(configmanager, 'get_prop', lambda _: "fake")
+    monkeypatch.setattr(configmanager, 'get_prop', getprop)
     with pytest.raises(SystemExit) as error:
         execute({"action": "test123"})
 

